src/themes/uclouvain/app/shared/promoter-download-url/promoter-file-download-url.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Input } from '@angular/core';
+import { NotificationsService } from 'src/app/shared/notifications/notifications.service';
+
+// This component is used to display a download URL for a file, and allows a manager to copy it to the clipboard
+@Component({
+    selector: 'ds-promoter-file-download-url',
+    templateUrl: './promoter-file-download-url.component.html'
+})
+export class PromoterFileDownloadUrlComponent {
+    @Input() downloadUrl: string;
+
+    constructor(private notificationsService: NotificationsService) {
+        //
+    }
+
+    copyUrlToClipboard(): void {
+        navigator.clipboard.writeText(this.downloadUrl).then(() => {
+            // Use DSpace alert system to give feedback to the user
+            this.notificationsService.success('The download URL has been copied to the clipboard.');
+        }).catch((error) => {
+            this.notificationsService.error('Error while copying the download URL to the clipboard.');
+        });
```

Comment:
for single instruction function, you don't need to use bracket. It could be more readable with better indentation
```typescript
navigator.clipboard.writeText(this.downloadUrl)
    .then(() =>  this.notificationsService.success('The download URL has been copied to the clipboard.'))
    .catch(_ => this.notificationsService.error('Error while copying the download URL to the clipboard.'));
```